docs/articles/expensify-classic/account-settings/Merge-Accounts.md

```diff
@@ -1,5 +1,33 @@
 ---
 title: Merge Accounts
-description: Merge Accounts
+description: How to merge two Expensify accounts and why this is useful.
 ---
-## Resource Coming Soon!
+
+# Overview
+
+Merging accounts allows you to combine two accounts. When you combine two accounts, all receipts, expenses, expense reports, invoices, bills, imported cards, secondary logins, co-pilots, and group policy settings will be combined into one account. 
+This can be useful if you start off with an account of your own but your organization creates a separate account for you. You can the track both personal and business expenses via one account. 
+
+# How to merge accounts
+Merging two accounts together is fairly straightforward. Let’s go over how to do that below:
+1. Navigate to expensify.com. 
+2. Log into the account you want to set as the Primary account.
+3. Navigate to Settings > Account > Account Details
```

Comment:
Some numbered steps end with periods and others do not. Should we pick one and stick with it?

Reply:
I think I fixed this.